content/cpp/concepts/math-functions/terms/cbrt/cbrt.md

```diff
@@ -37,3 +37,22 @@ int main() {
   // Output: The cube root of 27 is 3
 }
 ```
+
+
+## Codebyte Example
+
+We use the `cbrt` function from the `<cmath>` library to calculate the cube root of `x`.
```

Comment:
This codebyte is just a copy of the existing example in the entry. The codebyte should be different, and ideally it will show some other functionality or information that isn't present in the other example. Please create an original example for this codebyte.